backend/src/test/java/com/woowacourse/gongcheck/application/JobServiceTest.java

```diff
@@ -81,4 +84,17 @@ class JobServiceTest {
                 .isInstanceOf(NotFoundException.class)
                 .hasMessage("존재하지 않는 공간입니다.");
     }
+
+    @Test
+    void 공간과_섹션들과_작업들을_생성한다() {
+        Host host = hostRepository.save(Host_생성("1234"));
+        Space space = spaceRepository.save(Space_생성(host, "잠실"));
+        List<TaskRequest> tasks = List.of(new TaskRequest("책상 닦기"), new TaskRequest("칠판 닦기"));
+        List<SectionRequest> sections = List.of(new SectionRequest("대강의실", tasks));
+        JobCreateRequest jobCreateRequest = new JobCreateRequest("청소", sections);
+
+        Long savedJob = jobService.createJob(host.getId(), space.getId(), jobCreateRequest);
```

Comment:
```suggestion
        Long result = jobService.createJob(host.getId(), space.getId(), jobCreateRequest);
```
혹은 `savedJobId`가 더 직관적일 것 같아요 :)